src/Cache/ApcuCache.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PhpMyAdmin\MoTranslator\Cache;
+
+use PhpMyAdmin\MoTranslator\CacheException;
+use PhpMyAdmin\MoTranslator\MoParser;
+
+use function apcu_enabled;
+use function apcu_entry;
+use function apcu_exists;
+use function apcu_fetch;
+use function apcu_store;
+use function array_combine;
+use function array_keys;
+use function array_map;
+use function assert;
+use function function_exists;
+use function is_array;
+use function is_string;
+
+final class ApcuCache implements CacheInterface
+{
+    public const LOADED_KEY = '__TRANSLATIONS_LOADED__';
+
+    /** @var MoParser */
+    private $parser;
+    /** @var string */
+    private $locale;
+    /** @var string */
+    private $domain;
+    /** @var int */
+    private $ttl;
+    /** @var bool */
+    private $reloadOnMiss;
+    /** @var string */
+    private $prefix;
+
+    public function __construct(
+        MoParser $parser,
+        string $locale,
+        string $domain,
+        int $ttl = 0,
+        bool $reloadOnMiss = true,
+        string $prefix = 'mo_'
+    ) {
+        // @codeCoverageIgnoreStart
```

Comment:
I think we can cover this too, by adding two different matrix one without and one with the extension and also writing a test for it